### PR TITLE
Fixes fastify/help#442

### PR DESCRIPTION
**Pull request by luisorbaiceta**
Refactored code to new property validatorCompiler

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by mcollina — CHANGES_REQUESTED** [review]
The version in package.json must be updated as well https://github.com/fastify/example/blob/master/validation-messages/package.json#L19

**Comment by luisorbaiceta** [comment]
> The version in package.json must be updated as well https://github.com/fastify/example/blob/master/validation-messages/package.json#L19

Done. I have also updated tap version due to found vulnerabilities

File: validation-messages/package.json [hunk 1]
```diff
@@ -16,9 +16,9 @@
     "ajv-errors": "^1.0.1"
   },
   "devDependencies": {
-    "fastify": "^2.3.0",
+    "fastify": ">=3.0.0",
```
Comment (mcollina) [inline validation-messages/package.json]:
```suggestion
    "fastify": "^3.0.0",
```

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**